nuklear.h

```diff
@@ -22827,7 +22827,15 @@ nk_group_begin_titled(struct nk_context *ctx, const char *id,
         NK_ASSERT(y_offset);
         if (!x_offset || !y_offset) return 0;
         *x_offset = *y_offset = 0;
-    } else y_offset = nk_find_value(win, id_hash+1);
+    } else {
+        y_offset = nk_find_value(win, id_hash+1);
+        if (!y_offset) {
+            y_offset = nk_add_value(ctx, win, id_hash+1, 0);
+            NK_ASSERT(y_offset);
+            if (!y_offset) return 0;
+            *y_offset = 0;
+        }
+    }
```

Comment:
Good find, thanks a lot. Two things...

1. Make sure to update the change in the `src` directory too. Otherwise the next time the paq.sh file is run, we'll lose this change.
2. Also, what do you think about updating this pattern in both `nuklear_group.c` and `nuklear_list_view.c` too? Could probably run into the same issue there, as it's using a similar pattern.

Reply:
1. Ah, right, sorry! I just ported the change from the vendored file in my project, I didn't read up on how development actually goes here. Sorry. Do I need to _also_ edit `nuklear.h` or will that be handled automatically when you pack it up?
2. Yep, seems similar. Do you want the same change there as well, or do you have some nicer way to generalize it in mind?

Thanks!

Reply:
1. No need to appologize. Can be confusing. You don't absolutely need to edit nuklear.h. Anyone can run the script and it'll get updated eventually. Whichever is easier for you
3. Same way is fine :shrug:

Reply:
Done. I left it as smaller commits, but let me know if you prefer a squash. Or just squash when merging.